src/js/edu-benefits/1995/components/ServicePeriodView.jsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 
+/**
+ * Accepts YYYY-MM-DD and turns it into DD/MM/YYYY
+ *
+ * @param {String} date
+ * @return {String}
+ */
+function formatDisplayDate(date) {
```

Comment:
Ah, brilliant! I didn't see it before, so I just wrote a little regex. I'll reuse that.